src/libs/renderer/sdevice.cpp

```diff
@@ -453,7 +453,7 @@ bool DX9RENDER::Init()
         for (long i = 0; i < iCapBuffers; i++)
             aCaptureBuffers.push_back(new char[sizeof(uint32_t) * screen_size.x * screen_size.y]);
 
-        if (bWindow)
+        /*if (bWindow)
```

Comment:
This code just force-moves window to center of screen using non-portable WinAPI. I see no reason to keep this code, because SDL centers window by default.